src/config/dynamic-theme-fixes.config

```diff
@@ -17416,6 +17416,15 @@ IGNORE INLINE STYLE
 
 ================================
 
+myorca.*
+
+INVERT
+h1
```

Comment:
While you may seem it working on a specific page, it can break on other pages. I suggest using more specific selectors like `.m-banner h1` etc. Maybe instead of `INVERT` some `CSS` like `color: black !important` will do the job.